Code/Tools/FBuild/FBuildCore/Helpers/HtmlReport.cpp

```diff
@@ -0,0 +1,851 @@
+// HtmlReport
```

Comment:
This file is essentially the old `Report.cpp` file but without the common functionality - those have been extracted into the the new `Report.cpp`.